releases/release-1.31/release-notes/maps/pr-125828-map.yaml

```diff
@@ -1,6 +1,6 @@
 pr: 125828
 releasenote:
-  text: Bump the KubeletCgroupDriverFromCRI feature gate to beta and true by default.
+  text: Bumped the KubeletCgroupDriverFromCRI feature gate to beta and true by default.
```

Comment:
We might want to include some of the additional information from author's update release-note here, not sure why it wasn't picked up: https://github.com/kubernetes/kubernetes/pull/125828

```
Promoted CRI communication of cgroup driver mechanism, to beta.
The `KubeletCgroupDriverFromCRI` feature gate is now beta and enabled by default. This means that the kubelet queries the container runtime using CRI to discover the mechanism for cgroup management that the kubelet will use; if the container runtime doesn't support this, the kubelet falls back to loading this from its configuration file (you can also use the deprecated `--cgroup-driver` command line argument).
```